spec/src/utils/request-queue.js

```diff
@@ -321,7 +321,7 @@ describe('ConstructorIO - Utils - Request Queue', () => {
     });
 
     describe('Two Instances', () => {
-      it('Should send tracking requests using both instances if requests exist in storage and user is human', (done) => {
+      it('Should send tracking requests from both queues if requests exist in storage and user is human', (done) => {
```

Comment:
maybe reword to 'Should send tracking requests when using both queues...' or 'Should send tracking requests when using multiple queues...'  ?